content/blog/2017/2017-02-14-pipleine-editor-preview.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Say 'Hello' to the Blue Ocean Pipeline Editor"
+tags:
+- blueocean
+- editor
+- declarative
+- pipeline
+author: jamesdumay
+---
+
+link:/blog/2016/09/19/blueocean-beta-declarative-pipeline-pipeline-editor/[Back in September 2016 we announced the availability of the Blue Ocean beta]
+and the forthcoming Visual Pipeline Editor. We are happy to announce that you can try
+the Pipeline Editor preview release today.
+
+=== What is it?
+
+The Pipeline Editor is the simplest way for anyone wanting to get started with
+creating Pipelines in Jenkins. It's also a great way for advanced Jenkins users
+to start adopting pipeline. It allows developers to break up their pipeline into different
+ stages and parallelize tasks that can occur at the same time - graphically.
+ The rest is up to you.
+
+A pipeline you create or edit visually can produce a link://blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline] Jenkinsfile for you. This is a "round trip" editor which lets you load the pipeline back in later, for more editing. Not sure what a Jenkins Pipeline is? Why not check out our link:/doc/[new guided tour]!
+
+image::/images/blueocean/editor-1.png["The Editor", role="center"]
+
+=== Whats next?
+
+`Jenkinsfile`s belong in the Git repository so that it is versioned a long side
```

Comment:
The GitHub preview formatting is broken in this paragraph, did you test to see whether it works on the site?